.github/workflows/ci.yml

```diff
@@ -8,7 +8,7 @@ jobs:
   CI:
     strategy:
       matrix:
-        node: ["16.x", "18.x"]
+        node: ["18.x", "20.x", "22.x"]
```

Comment:
The reason for the change is that 16 is EOL'ed, and I believe it doesn't support `node:test`.

Also added 20 and 22 to make sure it works with them.

Reply:
`node:test` is much better in 22. It was more experimental in the previous versions.